psychopy/experiment/components/envelopegrating/__init__.py

```diff
@@ -31,6 +31,7 @@
               'envori':_translate('Envelope orientation'),
               'envphase':_translate('Envelope phase'),
               'moddepth':_translate('Envelope modulation depth'),
+              'power':_translate('Power to which envelope is rasied'),
```

Comment:
rasied = raised

Reply:
fixed.

Reply:
FWIW I wouldn't normally push for a recommit on a typo but this is in a piece of text that will be translated (e.g. into Japanese) and when the typo does get corrected it will show up as needing re-translation